DSA/AVL.java

```diff
@@ -0,0 +1,301 @@
+import java.util.*;
```

Comment:
import your packages explicitly instead of using '*'.

Reply:
okay Sir, i will take care of it next time.